src/patternfly/components/OptionsMenu/examples/options-menu.md

```diff
@@ -200,6 +200,7 @@ import './options-menu.css'
 | `.pf-m-top` | `.pf-c-options-menu` | Modifies to display the menu above the toggle. |
 | `.pf-m-align-right` | `.pf-c-options-menu__menu` | Modifies to display the menu aligned to the right edge of the toggle |
 | `.pf-m-expanded` | `.pf-c-options-menu` |  Modifies for the expanded state. |
+| `.pf-m-static` | `.pf-c-options-menu__menu` | Modifies to position the menu statically to support custom positioning. |
```

Comment:
I think the wording on the menu item is slightly clearer, "Modifies the menu to be statically positioned to support custom positioning." Though I don't love positioned-positioning but I can't come up with anything better.